ts/main.ts

```diff
@@ -51,12 +53,18 @@ const $exerciseDetailSection = document.querySelector(
   '#details-section',
 ) as HTMLElement;
 let $exercisesNodeList: NodeListOf<HTMLElement>;
-const $heart = document.querySelector('.fa-heart') as HTMLElement;
+let $favoritesNodeList: NodeListOf<HTMLElement>;
+const $detailsHeart = document.querySelector(
```

Comment:
Do these variables need to be global for your NodeLists? NodeLists are static so if you make changes to them it's going to be different anyways so have a global variable for it doesn't make much sense to me.

You should be able to create a const variable wherever you are querying for this.